tutorials/Tutorial1_Basic_QA_Pipeline.py

```diff
@@ -10,14 +10,19 @@
 # marvellous seven kingdoms.
 
 import logging
+
+# We configure how logging messages should be displayed and which log level should be used before importing Haystack.
+# Example log message:
+# INFO - haystack.utils.preprocessing -  Converting data/tutorial1/218_Olenna_Tyrell.txt
+logging.basicConfig(format="%(levelname)s - %(name)s -  %(message)s", level=logging.WARNING)
```

Comment:
I think `WARNING` is the default, right? If it's so we can remove the explicit level imho, or add a comment saying that this is the default anyway (same in the notebook)

Reply:
That's right. I added a comment that `WARNING` is the default.